db/seeds.rb

```diff
@@ -0,0 +1,17 @@
+require 'csv'
+require './app/models/station.rb'
+
+def load_station(data)
+  csv_text = File.read(data)
+  csv = CSV.parse(csv_text, :headers => true)
+  csv.each do |row|
```

Comment:
Looks like this is doing what you need it to, but check into [::foreach](http://ruby-doc.org/stdlib-2.0.0/libdoc/csv/rdoc/CSV.html#method-c-foreach). Will be able to accomplish the same task with fewer steps.